template-helpers/breaches.js

```diff
@@ -106,21 +62,28 @@ function lastAddedBreach(options) {
 function getFoundBreaches(args) {
   const locales = args.data.root.req.supportedLocales;
   let userBreaches = args.data.root.foundBreaches;
-
-  userBreaches = makeBreachCards(userBreaches, locales, false);
-
+  userBreaches = makeBreachCards(userBreaches, locales);
   userBreaches.cardType = "two-up drop-shadow";
   return userBreaches;
 }
 
-function getBreachArray(args) {
+function getLocalizedBreachValues(locales, breach) {
+  breach.AddedDate = prettyDate(breach.AddedDate, locales);
+  breach.PwnCount = localeString(breach.PwnCount,locales);
+  breach.DataClasses = dataClassesforCards(breach, locales);
+  return breach;
+}
+
+function getBreachArray(breaches, args) {
   const locales = args.data.root.req.supportedLocales;
-  let allBreaches = args.data.root.breaches;
+  breaches = JSON.parse(JSON.stringify(breaches));
```

Comment:
Again do we need `JSON.parse(JSON.stringify())` together?

Reply:
Same as previous comment.